src/garth_mcp_server/auth.py

```
@@ -0,0 +1,116 @@
+
+    if title_text != "Success" and not re.search(
+        r'ticket=([^"]+)"', client.last_resp.text
+    ):
+        print(f"Login failed. Page title: {title_text}", file=sys.stderr)
+        sys.exit(1)
+
+    try:
+        oauth1, oauth2 = sso._complete_login(client)
+        client.oauth1_token = oauth1
+        client.oauth2_token = oauth2
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)